src/table/table-adapter.class.ts

```diff
@@ -116,23 +127,47 @@ export class TableDomAdapter implements TableAdapter {
 	 * @param column index of the column
 	 */
 	getCell(row: number, column: number): HTMLTableCellElement {
-		const baseRow = this.getRow(row);
-		let baseCells = Array.from(baseRow.cells);
+		const col = this.getColumn(column);
 
-		// rows can have fewer total cells than the actual table
-		// the model pretends all rows behave the same (with colspans > 1 being N cells long)
-		// this maps that view to the HTML view (colspans > 1 are one element, so the array is shorter)
-		let realIndex = 0;
-		// i is only used for iterating the "cells"
-		for (let i = 0; i < column;) {
-			// skip the next N "cells"
-			i += baseCells[realIndex].colSpan;
-			// don't bump realIndex if i now exceeds the column we're shooting for
-			if (i > column) { break; }
-			// finally, increment realIndex (to keep it generally in step with i)
-			realIndex++;
+		return this.findCellInColumn(col, row).cell;
+	}
+
+	/**
+	 * Returns a column from the table, using the `id` and `headers` attributes
+	 *
+	 * See here for more detail these attributes: https://www.w3.org/TR/WCAG20-TECHS/H43.html
+	 *
+	 * @param column the index of the column
+	 */
+	getColumn(column: number): HTMLTableCellElement[] {
+		const firstHeader = Array.from(this.tableElement.rows[0].cells);
```

Comment:
`Array.from` has no IE support

![image](https://user-images.githubusercontent.com/14989804/69885688-5cdec680-12ac-11ea-8f95-199ff1004e48.png)

Reply:
That's fine, we have polyfills for that 👍